api/taskcluster/webhook.go

```diff
@@ -163,12 +163,18 @@ func GetCheckSuiteEventInfo(checkSuite github.CheckSuiteEvent, log shared.Logger
 			group.TaskGroupID = taskID
 		}
 
-		log.Debugf("Adding task: %s, id: %s, status: %s", run.GetName(), taskID, run.GetStatus())
+		log.Debugf("Adding task: %s, id: %s, conclusion: %s", run.GetName(), taskID, run.GetConclusion())
+
+		// Match the GitHub Check Runs API to the old Taskcluster status API.
```

Comment:
This is a bit off. `TaskInfo` is what Taskcluster API returns (see `GetTaskGroupInfo`). In the old world, we always call the API and deserialize the response (partially) ourselves. In the new world, if we were to call the API, the response would still be the same; however, we have enough information from the GitHub webhook payload to reconstruct `TaskInfo` without calling the API, so perhaps

```suggestion
		// Reconstruct Taskcluster TaskInfo from the check run without calling Taskcluster API.
```